01_science_technology_and_epistemology/01_science_technology_and_epistemology.jl

```diff
@@ -8,72 +8,175 @@ using InteractiveUtils
+Only after this, it is useful to formalize our knowledge, in order to be able to lay solid foundations and to continue advancing in our understanding.
 
 ## What is technology?
 
-Okay, technology is the thing that allows us as humans to transform reality. And here it is important to clarify that technology is not something "new". It has been with us since the very beginning of the human era, and it is really what makes us different from other animals. 
+Technology is the thing that allows us as humans to transform reality. And here it is important to clarify that technology is not something "new". 
+It has been with us since the very beginning of the human era, and it is really what makes us different from other animals. 
 
-How? Well, when we say that we use technology to transform the reality, we are referring to the fact that we do so through the *expansion of our capabilities*. From taking a long stick with a point to reach a fruit high up in a tree, passing through transforming an entire landscape by building a huge electric dam, to the creation of artificial intelligence that helps us solve our deepest puzzles. With technology we transform our *experience* in the reality in which we live.
+How? Well, when we say that we use technology to transform the reality, we are referring to the fact that we do so through the *expansion of our capabilities*.
+From taking a long stick with a point to reach a fruit high up in a tree, passing through transforming an entire landscape by building a huge electric dam, to the creation of artificial intelligence that helps us solve our deepest puzzles. 
+With technology we transform our *experience* in the reality in which we live.
 
-The current understanding of how knowledge is produced is very strange and even counterproductive for the development of humanity. Nowadays, there is a kind of *linear thinking* model in which we take for granted that the process of knowledge production is very definite: We first make academic and theoretical discoveries and only then, with all of that formalization made, we can start thinking of putting it to practical use in the real world. This is a completely wrong way of thinking and is holding us back enormously, causing important funds to be misused.
+The current understanding of how knowledge is produced is very strange and even counterproductive for the development of humanity. 
+Nowadays, there is a kind of *linear thinking* model in which we take for granted that the process of knowledge production is very definite: We first make academic and theoretical discoveries and only then, with all of that formalization made, we can start thinking of putting it to practical use in the real world. 
+This is a completely wrong way of thinking and is holding us back enormously, causing important funds to be misused.
```

Comment:
Take a more nuanced approach, or provide justification/examples for such bold claims

Reply:
I like it. I would change "most" by "many" as in the other sentence.